frontend/src/features/apartment/ApartmentDetailsPage.tsx

```diff
@@ -76,16 +102,144 @@ const ConfirmedPrice = ({confirmed}) => {
     );
 };
 
+interface UnconfirmedPricesDownloadModalProps {
+    apartment: IApartmentDetails;
+    isVisible: boolean;
+    setIsVisible;
+}
+const UnconfirmedPricesDownloadModal = ({apartment, isVisible, setIsVisible}: UnconfirmedPricesDownloadModalProps) => {
+    const [additionalInfo, setAdditionalInfo] = useState("");
+
+    return (
+        <Dialog
+            id="unconfirmed-prices-download-modal"
+            closeButtonLabelText=""
+            aria-labelledby=""
+            isOpen={isVisible}
+            close={() => setIsVisible(false)}
+            boxShadow
+        >
+            <Dialog.Header
+                id="unconfirmed-prices-download-modal__header"
+                title="Lataa Enimmäishinta-arvio"
+            />
+            <Dialog.Content>
+                <FormTextInputField
+                    id="input-additional_details"
+                    key="input-additional_details"
+                    label="Lisätietoja"
+                    size="large"
+                    value={additionalInfo}
+                    setFieldValue={setAdditionalInfo}
+                    tooltipText="Lisätietokenttään kirjoitetaan, jos laskelmassa on jotain erityistä, mitä osakkaan on syytä tietää. Kentän teksti lisätään tulostettavaan hinta-arvio PDF:ään."
+                />
+            </Dialog.Content>
+            <Dialog.ActionButtons>
+                <Button
+                    onClick={() => setIsVisible(false)}
+                    variant="secondary"
+                    theme="black"
+                    size="small"
+                >
+                    Sulje
+                </Button>
+                <Button
+                    theme="black"
+                    size="small"
+                    variant="secondary"
```

Comment:
This button shouldn't have the `variant="secondary"` attribute, as it's the main action button in the modal.

Reply:
Fixed :)